app/static/index.js

```diff
@@ -84,6 +84,10 @@ function buttonHandling() {
                 button.classList.remove("fa-regular");
                 button.classList.add("fa-solid");
             }
+            else if (isClicked) {
```

Comment:
In my opinion, the comment button feels better if it stays darkened on mouse hover if it's been clicked.

Reply:
So I think it will feel better without this 'if' statement

Reply:
Lets do a poll - I prefer the if statement handling